Makefile

```diff
@@ -0,0 +1,54 @@
+.PHONY: all build start stop restart logs
+
+APPNAME     := website
+VERSION     := 1.0.0
+TEAM        := tourainetech
+IMAGE       := $(TEAM)/$(APPNAME):$(VERSION)
+
+#Usefull in remote Docker host management
+#DOCKER_HOST ?= tcp://127.0.0.1:2375
+#DOCKER_CMD  := $(DOCKER_BIN) -H $(DOCKER_HOST)
+DOCKER_BIN  ?= docker
+WORKSPACE   ?= .
+DOCKER_CMD = $(DOCKER_BIN)
+
+
+NO_COLOR=\033[33;0m
+OK_COLOR=\033[33;32m
+ERR_COLOR=\033[33;31m
+INFO_COLOR=\033[33;01m
+
+all: build checkdb start
```

Comment:
what is checkdb target ? bad copy/paste ? Indeed `make` command is not working !

Reply:
yup done